mail_config.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$use_mail_registration = false; //remeber to configure mail() function in php if you change to true!!!
+$expiration_time = 60*60; //time in seconds
+
+$mail_subject = 'Verification code'; // Give the email a subject 
+$mail_message = '
+ 
+Thanks for signing up!
+Your account has been created, you can login after activating it using url below.
+ 
+Please click this link to activate your account:
+http://www.yourwebsite.com/verify.php?code=<registration_code/>
```

Comment:
Very minor thing, but using `https` might stop people leaving it as `http` and then potentially leaking the URL.